src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -63,11 +63,13 @@ The .NET Foundation licenses this file to you under the MIT license.
 
     <ItemGroup>
       <NetCoreAppNativeLibrary Include="System.Native" />
-      <NetCoreAppNativeLibrary Include="System.Globalization.Native" Condition="'$(StaticICULinking)' != 'true'" />
+      <!-- FIXME: The library is currently not available for iOS-like platforms -->
```

Comment:
Do we have an issue we could link to?

Reply:
https://github.com/dotnet/runtime/issues/82389